src/main/daml/Daml/Finance/Asset/Fungible.daml

```diff
@@ -25,7 +25,9 @@ type F = Factory
 type T = Fungible
 
 instance Fungible.HasImplementation T
--- | Implementation of a transferable, lockable, and fungible holding.
+-- | Implementation of a fungible token.
```

Comment:
```suggestion
-- | Implementation of a fungible holding.
```

I wonder if we should stick to `holding` as that is the terminology we use across the library

Reply:
yes, agree. also removed another place where we mention token.